scripts/start/index.js

```diff
@@ -0,0 +1,68 @@
+import execa from 'execa';
+import inquirer from 'inquirer';
+
+const applications = new Set([
+  '@ovh-ux/manager-hub',
+  '@ovh-ux/manager-dedicated',
+  '@ovh-ux/manager-cloud',
+  '@ovh-ux/manager-public-cloud',
```

Comment:
No need to add `@ovh-ux/manager-telecom` or `@ovh-ux/manager-web` ? As it is proposed in the questipns ?

Reply:
I think this array is only here to detect if we  have to ask for a region (cf : https://github.com/ovh/manager/pull/2415/files#diff-cdf6d8d06626995121777296083524dbR47)

Reply:
You're right! I'm using it to be able to determine if it's related to a `region`.